docs/data/material/customization/palette/AddingColorTokens.js

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
 import { createTheme, ThemeProvider } from '@mui/material/styles';
 import { blue } from '@mui/material/colors';
-import { Box, Stack } from '@mui/system';
```

Comment:
Also replacing this import from `@mui/system` to `@mui/material`. This didn't seem intentional, but correct me if I'm wrong